tests/api2/test_service.py

```diff
@@ -19,16 +19,21 @@ def test_non_silent_service_start_failure():
     # Starting UPS service with invalid configuration will fail, let's use this to our advantage to test
     # service startup failure error reporting.
     call("service.stop", "ups")
+
+    time.sleep(5)
+
     call("datastore.update", "services.ups", 1, {"ups_monuser": ""})
 
+    time.sleep(5)
```

Comment:
This test seems to spam Systemd. I could be wrong, I added those sleep and it looks to have stopped the random failures.

Reply:
We're sleeping for 15 seconds in this single test alone. We're not doing this correctly. I'm assuming we're sleeping so many times so that we don't trigger systemd to put this service into the failed list?

Reply:
Please remove the sleeps that were added to this method and I'm going to write another (proper) test that avoids all of this entirely.

Reply:
Yes, and we have encountered that issue with the cifs service in the past in other tests. 

When that test fails it is on line 45. It is like previous calls never ran.

Reply:
Oh, I missed your last comment I will say ever that then.

Reply:
I reverted that change